src/stream-analytics/azext_stream_analytics/commands.py

```diff
@@ -0,0 +1,77 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-locals
+from azure.cli.core.commands import CliCommandType
+from ._validators import validate_streaming_job_start
+
+
+def load_command_table(self, _):
+
+    from ._client_factory import cf_jobs
+    stream_analytics_jobs = CliCommandType(
+        operations_tmpl='azext_stream_analytics.vendored_sdks.streamanalytics.operations._streaming_jobs_operations#StreamingJobsOperations.{}',
+        client_factory=cf_jobs)
+    with self.command_group('stream-analytics job', stream_analytics_jobs, client_factory=cf_jobs) as g:
+        g.custom_command('create', 'create_stream_analytics_job', supports_no_wait=True)
+        g.custom_command('update', 'update_stream_analytics_job')
+        g.custom_command('delete', 'delete_stream_analytics_job', supports_no_wait=True)
+        g.custom_show_command('show', 'get_stream_analytics_job')
+        g.custom_command('list', 'list_stream_analytics_job')
+        g.custom_command('start', 'start_stream_analytics_job', validator=validate_streaming_job_start, supports_no_wait=True)
+        g.custom_command('stop', 'stop_stream_analytics_job', supports_no_wait=True)
```

Comment:
Add a wait command for command groups with `supports_no_wait=True` command.

Reply:
there are 4 subgroups which `has support_no_wait=True` command(`job`, `input`, `output`, `function`), however each of them reference the provisioningState of `job`. So I add `wait` command for `job` only. Could you please advice whether it is right ?

Reply:
If they use the same get api for job, then I think it is the right way.